### PR TITLE
Nicer footer, responsiveness issues fixed

### PR DESCRIPTION
**Pull request by ashocka**
- added recap section at the end of each part
- made all steps into H2
- hacked in a few fixes to have it look ok on different resolutions
- adjusted footer layout and style

### CODE REVIEW

File: 2-new-project/index.md [hunk 1]
```diff
@@ -111,8 +111,8 @@ Once you add your own text, save the file in Sublime (hint: Cmd + S is the keybo
 Hint: make sure Jekyll is running in the Terminal (remember <code>jekyll serve</code>?). If there's anything wrong with the code, the Terminal will also show you potential errors, so check your Terminal window whenever the website doesn't load.
 </div>
 
-# What did you learn?
-
+<div class="recap">
 Wow, you have your own website now! You created it by starting a new Jekyll project through the Terminal (`jekyll new my-website`) and then running the website with `jekyll serve`. You were then able to open the website in your web browser, and edit the title, email and description on the front page.
```
Comment (ialja) [inline 2-new-project/index.md]: When you wrap Markdown text in divs, the Markdown syntax is ignored, so you have to manually add `<code>` tags in place of backticks `

File: js/jquery.scripts.js [hunk 2]
```diff
@@ -5,4 +5,6 @@ $( document ).ready(function() {
 	var result = test.replace(/\:/g,':<hr />');      
 
 	$('.post-title').html(result);
+
+	$('.recap').prepend("<h2>What did you learn?</h2>");
```
Comment (ialja) [inline js/jquery.scripts.js]: Might just be me, but this might be a bit of an overkill :smile: 

Also, missing newline at the end of the file ^^

Comment (ashocka) [inline js/jquery.scripts.js]: The problem is that you can't seem to wrap a segment inside a div and keep using markdown inside it.

I wanted to create a recap segment with a different background so it stands out. I tried wrapping it into div and adding a "## What did you learn?" headline. Didn't work, so I opted out for inserting this H2 with jQuery.

Is there a better solution?

EDIT: Saw your comment above. Cool!

File: 2-new-project/index.md [hunk 3]
```diff
@@ -111,8 +111,9 @@ Once you add your own text, save the file in Sublime (hint: Cmd + S is the keybo
 Hint: make sure Jekyll is running in the Terminal (remember <code>jekyll serve</code>?). If there's anything wrong with the code, the Terminal will also show you potential errors, so check your Terminal window whenever the website doesn't load.
 </div>
 
-# What did you learn?
-
+<div class="recap">
+<h2>What did you learn?</h2>
 Wow, you have your own website now! You created it by starting a new Jekyll project through the Terminal (`jekyll new my-website`) and then running the website with `jekyll serve`. You were then able to open the website in your web browser, and edit the title, email and description on the front page.
```
Comment (ialja) [inline 2-new-project/index.md]: Wrap jekyll commands in `<code>` tags in this paragraph and you win :)

Comment (ashocka) [inline 2-new-project/index.md]: uh. sorry about that.

**Pull request merged**